doc/docs/openapi.md

```diff
@@ -53,9 +54,12 @@ OpenAPIDocsInterpreter.toOpenAPI(List(addBook, booksListing, booksListingByGenre
 The openapi case classes can then be serialised, either to JSON or YAML using [Circe](https://circe.github.io/circe/):
 
 ```scala mdoc:silent
+import io.circe.syntax.EncoderOps
+import sttp.tapir.openapi.circe._
 import sttp.tapir.openapi.circe.yaml._
 
 println(docs.toYaml)
+println(docs.asJson)
```

Comment:
`.asJson` only coverts to a `Json` object, this isn't yet serialised to a string, right?

Reply:
yes, but println() automatically convert it

Reply:
let's maybe add an explicit example how to serialise to a string (if I remember correctly, using `Printer`)

Reply:
corrected